modules/HomePage/components/TaskList/TaskList.tsx

```diff
@@ -140,7 +143,7 @@ const TaskList: React.FC<TaskListProps> = ({
                   Priority: {task.priority}
                 </Text>
                 <Text className={classes.taskDueDate}>
-                  Due Date: {new Date(task.dueDate).toDateString()}
+                  Due Date: {new Date(task.dueDate ?? "").toDateString()}
```

Comment:
In the event that task.dueDate does not exist, wouldn't new Date("") just give you back todays data?

Reply:
Pushed a commit to address this in a better way